Index: app/src/main/java/bobrchess/of/by/belaruschess/view/activity/impl/AbstractActivity.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/bobrchess/of/by/belaruschess/view/activity/impl/AbstractActivity.kt	(date 1587933946000)
+++ app/src/main/java/bobrchess/of/by/belaruschess/view/activity/impl/AbstractActivity.kt	(date 1587933918065)
@@ -46,7 +46,7 @@
         registerReceiver(broadcastReceiver, internetFilter)
     }
 
-    override fun showAlertDialog(@StringRes title: Int, @StringRes message: Int, @StringRes buttonText: Int, cancelable: Boolean) {
+    override fun showAlertDialog(title: String, message: String, buttonText: String, cancelable: Boolean) {
         if (dialog != null) {
             dialog?.dismiss()
         }
Index: app/src/main/java/bobrchess/of/by/belaruschess/view/activity/impl/ConfirmationDialog.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/bobrchess/of/by/belaruschess/view/activity/impl/ConfirmationDialog.kt	(date 1587933946000)
+++ app/src/main/java/bobrchess/of/by/belaruschess/view/activity/impl/ConfirmationDialog.kt	(date 1587933918017)
@@ -3,25 +3,20 @@
 import android.app.Dialog
 import android.content.Context
 import android.os.Bundle
-import android.support.annotation.StringRes
 import android.support.v4.app.DialogFragment
 import android.support.v7.app.AlertDialog
-import android.content.DialogInterface
-import android.view.View
 
 
 class ConfirmationDialog : DialogFragment() {
 
     var alertDialog: AlertDialog? = null
     private lateinit var listener: ConfirmationListener
-    private var title: Int = 0
-    private var message: Int = 0
-    private var buttonText: Int = 0
+    private var title: String? = null
+    private var message: String? = null
+    private var buttonText: String? = null
 
-    //private var isCancelable: Boolean = false
 
-
-    fun setData(@StringRes title: Int, @StringRes message: Int, @StringRes buttonText: Int, cancelable: Boolean): DialogFragment {
+    fun setData(title: String?, message: String?, buttonText: String?, cancelable: Boolean): DialogFragment {
         this.title = title
         this.message = message
         this.buttonText = buttonText
Index: app/src/main/java/bobrchess/of/by/belaruschess/view/activity/BaseContractView.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/bobrchess/of/by/belaruschess/view/activity/BaseContractView.java	(date 1587933946000)
+++ app/src/main/java/bobrchess/of/by/belaruschess/view/activity/BaseContractView.java	(date 1587933918092)
@@ -7,7 +7,7 @@
  */
 
 public interface BaseContractView {
-    void showAlertDialog(@StringRes int title, @StringRes int message, @StringRes int buttonText, boolean cancelable);
+    void showAlertDialog(String title, String message, String buttonText, boolean cancelable);
     void dismissAlertDialog();
     void showToast(Integer resId);
     void showToast(String message);
